include/Webservices/ConvertLead.php

```diff
@@ -264,4 +266,73 @@ function vtws_updateConvertLeadStatus($entityIds, $leadId, $user) {
 	}
 }
 
+function vtws_createEntities($entityIds) {
+	global $adb,$current_user;
+
+	if(isset($entityIds['Accounts']))
+		$accountid = vtws_getIdComponents($entityIds['Accounts'])[1];
+	if(isset($entityIds['Accounts']))
+		$contactid = vtws_getIdComponents($entityIds['Contacts'])[1];
+
+	$bmapname = 'LeadConversion';
+	$cbMapid = GlobalVariable::getVariable('BusinessMapping_'.$bmapname, cbMap::getMapIdByName($bmapname));
+	if ($cbMapid) {
+		$cbMap = cbMap::getMapByID($cbMapid);
+		$modules = $cbMap->ModuleSetMapping()->getFullModuleSet();
+
+		foreach ($modules as $module) {
+			if($module != "Potentials") {
```

Comment:
This will cause problems only if the user creates Account2Contacts or Contact2Accounts maps. Either way we should prevent that (TBH I didn't think of that)